app/assets/stylesheets/article-show.scss

```diff
@@ -99,41 +99,23 @@ article {
 
```

Comment:
On the show page, the "offscreen" additional articles are actually showing up due to a 1px change.

<img width="468" alt="Screen Shot 2020-02-13 at 6 45 45 PM" src="https://user-images.githubusercontent.com/3102842/74488849-734ef400-4e91-11ea-8764-908819965ac9.png">

Basically when the height of the window changes, flexbox cleverly knows how to cut off whole elements instead of clipping them, and it does it this way and it's causing this funny visual thing.